dist/.eslintrc.cjs

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+	root: true,
+
+	parserOptions: {
+		ecmaVersion: 5,
+		sourceType: "script"
+	},
+
+	globals: {
+		define: false,
+		module: true,
+		Symbol: false
+	},
+
+	overrides: [
+		{
+			files: "jquery{,.slim}.min.js",
+
+			parserOptions: {
+				ecmaVersion: 5,
+				sourceType: "script"
+			}
+		},
+
+		{
+			files: "jquery{,.slim}.js",
+			extends: "../.eslintrc-browser.cjs",
+
+			rules: {
+
+				// That is okay for the built version
+				"no-multiple-empty-lines": "off",
+
+				// When custom compilation is used, the version string
+				// can get large. Accept that in the built version.
+				"max-len": "off",
+				"one-var": "off"
+			}
+		},
+
+		{
+			files: ".eslintrc.cjs",
```

Comment:
Hmm, so we are linting this file. Why doesn’t it complain about sloppy mode?

Reply:
Yeah, I'm not sure what's wrong. It's linted, but it doesn't always complain about no 'use strict'; I'm looking into it.

Reply:
It is possible it would only complain if you have at least a single function in the file.

Or maybe this is the reason: https://github.com/jquery/jquery/pull/5297/files#r1259812907